cse/writer/ArticleMappingWriter.py

```diff
@@ -37,16 +40,16 @@ def printHeader(self, template=None):
             self.__writer.writerow(["article_id", "arcticle_url"])
         else:
             self.__writer.writerow(template)
-        self.__file.flush()
 
 
-    def printData(self, data):
-        for i, articleUrl in enumerate(data):
-            self.__writer.writerow([
-                str(i),
-                articleUrl,
-            ])
-        self.__file.flush()
+    def mapToId(self, origArticleId, articleUrl):
```

Comment:
Do you really want to rely on the `origArticleId` ? (Keep in mind the old API has no original id))

Reply:
for missing ids, i replaced them with the url, but we can change it back to only considering the urls

Reply:
It's up to you to change it or leave it. I think we won't run in problem because of this

Reply:
Keep it. 😄